python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -1254,6 +1261,9 @@ def report_asset_check(
             },
         )
 
+    def report_extra_message(self, payload: Any):
+        self._write_message(Method.report_extra_message, {"payload": payload})
```

Comment:
_Disclaimer - Saw this PR which is super useful for my team and decided to test it out, so forgive me if I've jumped the gun on it or am just misunderstanding something!_

I believe Method is being accessed like an enum here when it is actually a Literal. It is throwing the following error:

`Error: AttributeError: '_SpecialForm' object has no attribute 'report_extra_message'`

When I replace it with the string "report_extra_message", everything works as expected.

Reply:
ah yep just missed this spot when switching from `Enum` to `Literal` in the PR this is stacked no